e2e/src/acceptanceTest/kotlin/maru/e2e/CliqueToPosTest.kt

```diff
@@ -63,145 +66,173 @@ class CliqueToPosTest {
     @JvmStatic
     fun beforeAll() {
       qbftCluster.before()
-      maru.start()
     }
 
     @AfterAll
     @JvmStatic
     fun afterAll() {
       qbftCluster.after()
-      maru.stop()
     }
-  }
 
-  private val log: Logger = LogManager.getLogger(CliqueToPosTest::class.java)
-  private val besuFollowerExecutionEngineClient = createExecutionClient("http://localhost:9550")
-  private val nethermindFollowerExecutionEngineClient =
-    createExecutionClient(
-      "http://localhost:10550",
-      TestEnvironment.jwtConfig,
-    )
-  private val erigonFollowerExecutionEngineClient =
-    createExecutionClient(
-      "http://localhost:11551",
-      TestEnvironment.jwtConfig,
-    )
-  private val geth1ExecutionEngineClient = createExecutionClient("http://localhost:8561", TestEnvironment.jwtConfig)
-  private val geth2ExecutionEngineClient = createExecutionClient("http://localhost:8571", TestEnvironment.jwtConfig)
-  private val gethSnapServerExecutionEngineClient =
-    createExecutionClient("http://localhost:8581", TestEnvironment.jwtConfig)
-  private val gethExecutionEngineClients =
-    mapOf(
-      "follower-geth" to geth1ExecutionEngineClient,
-      "follower-geth-2" to geth2ExecutionEngineClient,
-      "follower-geth-snap-server" to gethSnapServerExecutionEngineClient,
-    )
-  private val followerExecutionEngineClients =
-    mapOf(
-      "follower-besu" to besuFollowerExecutionEngineClient,
-      "follower-erigon" to erigonFollowerExecutionEngineClient,
-      "follower-nethermind" to nethermindFollowerExecutionEngineClient,
-    ) + gethExecutionEngineClients
+    private fun createExecutionClient(
+      eeEndpoint: String,
+      jwtConfig: Optional<JwtConfig> = Optional.empty(),
+    ): Web3JExecutionEngineClient = Web3JExecutionEngineClient(createWeb3jClient(eeEndpoint, jwtConfig))
+
+    private val log: Logger = LogManager.getLogger(CliqueToPosTest::class.java)
+    private val besuFollowerExecutionEngineClient = createExecutionClient("http://localhost:9550")
+    private val nethermindFollowerExecutionEngineClient =
+      createExecutionClient(
+        "http://localhost:10550",
+        TestEnvironment.jwtConfig,
+      )
+    private val erigonFollowerExecutionEngineClient =
+      createExecutionClient(
+        "http://localhost:11551",
+        TestEnvironment.jwtConfig,
+      )
+    private val geth1ExecutionEngineClient = createExecutionClient("http://localhost:8561", TestEnvironment.jwtConfig)
+    private val geth2ExecutionEngineClient = createExecutionClient("http://localhost:8571", TestEnvironment.jwtConfig)
+    private val gethSnapServerExecutionEngineClient =
+      createExecutionClient("http://localhost:8581", TestEnvironment.jwtConfig)
+    private val gethExecutionEngineClients =
+      mapOf(
+//        "follower-geth" to geth1ExecutionEngineClient,
```

Comment:
why did you remove the follower-geth node?

Reply:
Because Geth 1.14+ won't sync with Clique, so it won't switch. I think it should sync from scratch though, so let me try and add it to the full sync test